scripts/windows/deploy.cmd

```diff
@@ -59,8 +59,23 @@ REM
     set machine=%1
     set rdst=\\%machine%\%rdst_dir%
     @mkdir %rdst%
-    xcopy /F /Y /S %src_dir% %rdst%
-    SCHTASKS /CREATE /S %machine% /RU SYSTEM /SC ONLOGON /TN %deploy_name% /TR "%ldst_dir%\start.cmd" /V1 /F
+    CALL .\scripts\windows\7z.exe a send.7z %src_dir%
+    mkdir send
+    move send.7z send
+    COPY /Y .\scripts\windows\7z.exe send
+    COPY /Y .\scripts\windows\7z.dll send
+    (
+        ECHO cd /d %%~dp0
+        ECHO CALL .\7z.exe x -y send.7z
+        ECHO del send.7z
+        ECHO del 7z.exe
+        ECHO del 7z.dll
+        ECHO del unzip.cmd
+    )  > .\send\unzip.cmd
+    xcopy /F /Y /S send %rdst%\..
+    SCHTASKS /CREATE /S %machine% /RU SYSTEM /SC ONLOGON /TN unzip /TR "%ldst_dir%\..\unzip.cmd" /V1 /F
```

Comment:
Thx for reminding me. Btw, I think deleting is enough. If I set ONCE, the usage requires me to set the \ST (start time).